vocabularies/trs.ttl

```diff
@@ -235,36 +236,38 @@ trs:UTCTime
   skos:prefLabel "Coordinated Universal Time"@en ;
   skos:topConceptOf <http://linked.data.gov.au/def/trs> ;
 .
-
 trs:ThousandsOfYearsAgo
   a skos:Concept ;
   a time:TRS ;
-  dct:source <https://www.bipm.org/en/bipm-services/timescales/> ;
-  skos:altLabel "ka"@en ;
   skos:broader trs:BeforePresent ;
   skos:inScheme <http://linked.data.gov.au/def/trs> ;
   skos:definition "Temporal position expressed numerically scaled in thousands of years increasing backwards relative to 1950. Calibration to 1950 not always significant or done, except when carbon dating methods are used."@en ;
   skos:prefLabel "Thousands of Years Ago"@en ;
 .
-
 trs:MillionsOfYearsAgo
   a skos:Concept ;
   a time:TRS ;
-  dct:source <https://www.bipm.org/en/bipm-services/timescales/> ;
   skos:altLabel "Ma"@en ;
   skos:broader trs:BeforePresent ;
   skos:inScheme <http://linked.data.gov.au/def/trs> ;
   skos:definition "Temporal position expressed numerically scaled in millions of years increasing backwards relative to 1950. Calibration to 1950 usually not significant or done."@en ;
   skos:prefLabel "Millions of Years Ago"@en ;
 .
-
 trs:BillionsOfYearsAgo
   a skos:Concept ;
   a time:TRS ;
-  dct:source <https://www.bipm.org/en/bipm-services/timescales/> ;
   skos:altLabel "Ga"@en ;
   skos:broader trs:BeforePresent ;
   skos:inScheme <http://linked.data.gov.au/def/trs> ;
   skos:definition "Temporal position expressed numerically scaled in billions of years increasing backwards relative to 1950. Calibration to 1950 usually not significant or performed."@en ;
   skos:prefLabel "Billions of Years Ago"@en ;
 .
+trs:BeforePresentCalibrated
+  a skos:Concept ;
+  a time:TRS ;
+  skos:altLabel "BP-Cal"@en ;
+  skos:broader trs:BeforePresentCalibrated ;
```

Comment:
Should this be skos:broader trs:BeforePresent? i.e. is BeforePresentCalibrated a specific type of BeforePresent?

Reply:
It should be BeforePresent instead of BeforePresentCalibrated

Reply:
correct to skos:broader trs:BeforePresent